clients/python/test/test_osparc/test_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
```

Comment:
in all tests add the following headers

```python
# pylint: disable=protected-access
# pylint: disable=redefined-outer-name
# pylint: disable=too-many-arguments
# pylint: disable=unused-argument
# pylint: disable=unused-variable
```
These are the defaults for all testing files.

Reply:
done